### PR TITLE
Fixes #3. Adds an Open Notifications worker pod.

### LINKED ISSUE #3: Open Notificaties background workers deployment is missing
The chart deploys RabbitMQ & the web service for ON, but the Celery worker for ON which actually sends the notifications to the subscriptions is missing.

Ansible equivalent is: https://github.com/open-zaak/ansible-collection/blob/main/roles/open_notificaties_k8s/templates/celery.yml.j2

### PR DESCRIPTION
**Pull request by andyverberne**
Fixes #3 

- Adds an Open Notifications worker pod
- Contains the changes made in Pull request #4 
- Removes the Chart.lock files which should not be part of the package (I think)

### CODE REVIEW

**Comment by andyverberne** [comment]
I just realised that I have incorrectly moved some items from secret to configMap. I will fix this asap.

File: charts/open-notificaties/templates/deployment.yaml [hunk 1]
```diff
@@ -85,3 +85,61 @@ spec:
...
       {{- end }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "open-notificaties.workerFullname" . }}
+  labels:
+  {{- include "open-notificaties.workerLabels" . | nindent 4 }}
+spec:
+  replicas: 1
```
Comment (sergei-maertens) [inline charts/open-notificaties/templates/deployment.yaml:96]: this should be configurable to scale the number of workers depending on workload

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: charts/open-notificaties/templates/deployment.yaml (hunk 1 above)
Comment (andyverberne) [reply charts/open-notificaties/templates/deployment.yaml:96]: I thought about that. Are you sure that if there are 2 or more workers, messages of the queue are not delivered more than once? I will make this scalable.

**Comment by andyverberne** [comment]
Worker pod is now scalable

**Review by sergei-maertens — APPROVED** [review]

File: charts/open-notificaties/templates/deployment.yaml (hunk 1 above)
Comment (sergei-maertens) [reply charts/open-notificaties/templates/deployment.yaml]: yes - the broker keeps track of them & the default deployment with Ansible fires up 3 replicas on both Docker-on-vm and Kubernetes ;)

**Pull request merged**